src/System.Net.Http.WinHttpHandler/src/System.Net.Http.WinHttpHandler.msbuild

```diff
@@ -15,6 +15,7 @@
     <CompileItem Include="$(CommonPath)\Interop\Windows\winhttp\Interop.winhttp.cs" />
     <CompileItem Include="$(CommonPath)\Microsoft\Win32\SafeHandles\SafeHandleZeroOrMinusOneIsInvalid.cs" />
     <CompileItem Include="$(CommonPath)\System\StringExtensions.cs" />
+    <CompileItem Include="$(CommonPath)\System\Net\HttpKnownHeaderNames.cs" />
```

Comment:
FWIW, the System.Net.Http.dll binary already drags in this class `HttpKnownHeaderNames.cs`.  Now this change will drag in another copy for the System.Net.Http.WinHttpHandler.dll as well.

But won't the compiler assist in this in some way by figuring out what is really needed and trim out the rest?